update_keb_installation.py

```diff
@@ -0,0 +1,124 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Update Kebechet installation details in DB during webhook workflows."""
+
+import os
+from tempfile import TemporaryDirectory
+from contextlib import contextmanager
+
+
+import yaml
+import json
+import requests
+from github import Github
+
+from thoth.storages import GraphDatabase
+from thamos.cli import _load_files
+from thoth.common import logging
+
+from thoth.workflow_helpers import __service_version__
+
+_SLUG = json.loads(os.getenv("GITHUB_SLUG") or "null")
+_ACCESS_TOKEN = os.getenv("KEBECHET_ACCESS_TOKEN")
+
+_APP_NAME = "KEBECHET"
+_GITHUB_URL = "https://github.com"
+
+db = GraphDatabase()
+db.connect()
+
+_LOGGER = logging.getLogger("thoth.update_kebechet_installation")
+_LOGGER.info("Thoth workflow-helpers task: update kebechet installation details v%s", __service_version__)
+
+
+@contextmanager
+def cwd(path: str):
+    """Change working directory in a push-pop manner with context manager."""
+    previous_dir = os.getcwd()
+    try:
+        os.chdir(path)
+        yield previous_dir
+    finally:
+        os.chdir(previous_dir)
+
+
+def _raw_from_github(project, branch, access_token, file_name):
+    headers = {"Authorization": f"token {_ACCESS_TOKEN}"}
+    url = f"https://raw.githubusercontent.com/{project}/{branch}/{file_name}"
+    return requests.get(url, headers=headers)
+
+
+def update_keb_installation():
+    """Load files and pass them to storages update function."""
+    if _SLUG is None:
+        _LOGGER.info("No slug present, continuing to next step in task.")
+        return
+
+    g = Github(_ACCESS_TOKEN)
+    repo = g.get_repo(_SLUG)
+    try:
+        r = _raw_from_github(_SLUG, repo.default_branch, _ACCESS_TOKEN, ".thoth.yaml")
+    except Exception:
+        _LOGGER.info("No .thoth.yaml file found, marking repo as inactive in DB")
+        db.update_kebechet_github_installations_on_is_active(_SLUG)
+
+    with TemporaryDirectory() as repo_path, cwd(repo_path):
+        thoth_config = yaml.load(r.content)
+        requirements_format = thoth_config["requirements_format"]
+        if requirements_format == "pipenv":
+            try:
+                pipfile_r = _raw_from_github(_SLUG, repo.default_branch, _ACCESS_TOKEN, "Pipfile")
+                with open("Pipfile", "wb") as f:
+                    f.write(pipfile_r.content)
+            except Exception:
+                _LOGGER.warn("pipenv requirements specified but no Pipfile found.")
```

Comment:
`warn` is depracated:

```suggestion
                _LOGGER.warning("pipenv requirements specified but no Pipfile found.")
```